ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -90,6 +90,17 @@
     "metadata.json",
 ]
 
+INCLUDED_FILE_EXTENSIONS = [".py"]
+
+
+def should_include_file(filename: str) -> bool:
+    """Check if a file should be included based on its extension.
+
+    This is used by both the serialization logic and the push API to ensure
+    only Python files are included in workflow artifacts.
```

Comment:
DevinAI apply this suggestion:
```suggestion
    This is used by both the serialization logic and the push API to ensure
    consistency in which files are included in workflow artifacts.
```